Assets/Global/Scripts/Menu/PauseLogic.cs

```diff
@@ -52,7 +54,29 @@ public void QuitGame()
         Menu.SetActive(!Menu.activeSelf);
         // Upgrades.SetActive(!Upgrades.activeSelf);
         Time.timeScale = 1f;
-        SceneManager.LoadScene("Menu");
+        //SceneManager.LoadScene("Menu");
+
+        if (GetCurrentSceneName() == "PARKOUR_1" || GetCurrentSceneName() == "PARKOUR_2" || GetCurrentSceneName() == "PARKOUR_3")
+        {
+            UILogic.FadeToScene("Loading", fadeImage, this);
+        }
+        else if (GetCurrentSceneName() == "ENTRANCE_1")
+        {
+            SceneManager.LoadScene("Menu");
+
+        }
+    }
+    public string GetCurrentSceneName()
```

Comment:
i think this code is copied from somewhere else in the code base, is there a way to not have duplicated code but rather use that method?